src/pages/VolunteersJobListingFS.page

```diff
@@ -159,6 +168,7 @@
                     	if (x == '') x = JobShiftDateTimeString('{!shift.Start_Date_Time__c}', '{!shift.Duration__c}');
                         OpenSignUpDlg(event.clientX, event.clientY, '{!job.Id}', '{!shift.Id}', '{!JSENCODE(job.Name)}', x); 
                         return false;"
+                        style="{!IF(job.External_Signup_Url__c == null, '', 'display: none;')}"
```

Comment:
rather than setting the style to display:none, couldn't you have just added your logic to the rendered formula in the surrounding outputPanel.  seems like that would be more consistent and readable with the existing code.